rasa/core/training/dsl.py

```diff
@@ -412,10 +415,12 @@ def add_checkpoint(self, name: Text, conditions: Optional[Dict[Text, Any]]) -> N
         )
         intent_name = utterance.intent.get("name")
         if intent_name not in self.domain.intents:
-            warnings.warn(
+            raise_warning(
                 f"Found unknown intent '{intent_name}' on line {line_num}. "
                 "Please, make sure that all intents are "
-                "listed in your domain yaml."
+                "listed in your domain yaml.",
+                UserWarning,
+                docs="https://rasa.com/docs/rasa/core/domains/",
```

Comment:
so here we give full URL to the doc compared to relative one before?
What's the desired way?

Reply:
fixed this, it is full URL now that we have constants (before it was relative, which is great - now we can also link to idk, some tensorflow issue or something)